contentwarehouse/snippets/search_documents_sample_test.py

```diff
@@ -0,0 +1,44 @@
+# # Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+
+from contentwarehouse.snippets import search_documents_sample
+
+from google.cloud import resourcemanager
+
+project_id = os.environ["GOOGLE_CLOUD_PROJECT"]
+location = "us"  # Format is 'us' or 'eu'
+document_query_text = "document"
+
+
+def test_search_documents(capsys):
+    project_number = get_project_number(project_id)
+    search_documents_sample.search_documents_sample(
+        project_number=project_number,
+        location=location,
+        document_query_text=document_query_text,
+    )
+    out, _ = capsys.readouterr()
+
+    assert "document" in out
+
+
+def get_project_number(project_id: str) -> str:
```

Comment:
This method looks like it's a duplicate of the one that exists in `quickstart_sample_test.py`. Fine to leave it as is, but if you can consider refactoring it so there's only one occurrence for better maintenance.

Reply:
It is a duplicate. Would it be reasonable to make a `utilities.py` file or something along those lines?